### DISCUSSION TITLE
Multiple streams?

### DISCUSSION THREAD

**Discussion by randolf** [obsproject/obs-studio#7615] [Development Questions] [answered]
I'm wondering if there are plans in the works for supporting the configuration of multiple streams.  At present, in the "Streams" section under "Settings" only one stream can be selected.  If it were possible to instead start with an empty list of streams, and have some Add / Edit / Enable / Disable / Remove buttons for managing them, this would be great for creators who have a need to stream to multiple networks and also multiple channels simultaneously on the same network.

I really like the way OBS Studio keeps pushing the stream output onto YouTube during a temporary network outage (or when the computer switches from a wireless to a wired connection) and the audience notices only a short pause (or doesn't even notice at all).  Having multiple stream outputs configurable within OBS Studio would be an amazing feature that would eliminate the need for messing around with RTSP or hosted third party re-streaming services (both of these options add complexity for creators, which count as additional potential points of failure).  I've also heard many YouTube creators ask about this because they want to stream simultaneously to YouTube, Twitch, etc.

Thanks.

**Comment by Fenrirthviti** [comment]
https://github.com/obsproject/rfcs/pull/39

**Comment by tytan652** [comment, marked as answer]
[RFC 39](https://github.com/obsproject/rfcs/pull/39) is talking about overhauling services which is a required step to add multi-streaming in the future.

So yes, there is plans to make plans for multi-stream.

**Reply by randolf** [reply]
Thank you.  This is excellent!

**Comment by v4u6h4n** [comment]
Given that #9220 has been closed, just wondering if is this looking more like a long term goal now?

**Reply by tytan652** [reply]
This is a goal to reach, it has been always a very slow process.

I need to re-write the Service Overhaul RFC once the service and streaming codebase is done with Amazon/Twitch changes that add more stuff to took in account hoping that no other actor will make the situation more complicated.
